Dockerfile.redhat

```diff
@@ -248,6 +248,9 @@ COPY .bazelrc WORKSPACE /ovms/
 RUN ln -s /usr/lib64 /usr/lib/x86_64-linux-gnu
 COPY external /ovms/external/
 COPY third_party /ovms/third_party/
+# This path is required for namespace to setup Python dependencies for testing the binding
+COPY src/BUILD /ovms/src/BUILD
```

Comment:
Are you sure we cannot move this below? I think we need those files in line 273, right?

Reply:
We could move it closer to 273 if you think it's more suitable. Line 273 requires those files in place.